Server/mods/deathmatch/logic/luadefs/CLuaResourceDefs.cpp

```diff
@@ -54,7 +54,7 @@ void CLuaResourceDefs::LoadFunctions()
         {"getResourceLoadFailureReason", getResourceLoadFailureReason},
         {"getResourceLastStartTime", getResourceLastStartTime},
         {"getResourceLoadTime", getResourceLoadTime},
-        {"getResourceName", getResourceName},
+        {"getResourceName", ArgumentParser<GetResourceName>},
```

Comment:
This makes the function backward incompatible. Because there're many scripts that expect this function returns false in case of failure. You should you `ArgumentParserWarn<false, GetResourceName>` here.

Reply:
Thank you.